versions.properties

```diff
@@ -130,483 +102,30 @@ version.io.jsonwebtoken..jjwt=0.9.1
-##                            # available=2.22.5
-##                            # available=2.22.6
-##                            # available=2.22.7
-##                            # available=2.23.0
-##                            # available=2.24.0
-##                            # available=2.25.0
-##                            # available=2.26.0
-##                            # available=2.27.0
-##                            # available=2.28.0
-##                            # available=2.29.0
-##                            # available=2.30.0
-##                            # available=2.31.0
-##                            # available=2.32.0
-##                            # available=2.33.0
-##                            # available=2.34.0
-##                            # available=2.35.0
-##                            # available=2.36.0
-##                            # available=2.37.0
-##                            # available=2.37.1
-##                            # available=2.38.0
-##                            # available=2.39.0
-##                            # available=2.40.0
-##                            # available=2.41.0
-##                            # available=2.41.1
-##                            # available=2.42.0
-##                            # available=2.43.0
-##                            # available=2.44.0
-##                            # available=2.45.0
-##                            # available=2.46.0
-##                            # available=2.47.0
-##                            # available=2.48.0
-##                            # available=2.48.1
-##                            # available=2.49.0
-##                            # available=2.50.0
-##                            # available=2.50.1
-##                            # available=2.51.0
-##                            # available=2.52.0
-##                            # available=2.52.1
-##                            # available=2.53.0
-##                            # available=2.54.0
-##                            # available=2.55.0
-##                            # available=2.56.0
-##                            # available=2.57.0
-##                            # available=2.58.0
-##                            # available=2.59.0
-##                            # available=2.59.1
-##                            # available=2.60.0
-##                            # available=2.61.0
-##                            # available=2.62.0
-##                            # available=2.62.1
-##                            # available=2.62.2
-##                            # available=2.63.0
-##                            # available=2.64.0
-##                            # available=2.65.0
-##                            # available=2.66.0
-##                            # available=2.67.0
-##                            # available=2.68.0
-##                            # available=2.69.0
-##                            # available=2.70.0
+version.com.amazonaws..aws-android-sdk-s3=2.71.0
```

Comment:
FYI, there is a deprecation warning, which I didn't have a chance to fix:

![image](https://github.com/airbnb/lottie-android/assets/36954793/1a8cfbc0-52cc-49d6-93e8-4b6761d67d6c)

I don't have a possibility to run these tests locally :/